src/Integration/Aws/SimpleS3/src/SimpleS3Client.php

```diff
@@ -47,6 +55,114 @@ public function has(string $bucket, string $key): bool
         return $this->objectExists(['Bucket' => $bucket, 'Key' => $key])->isSuccess();
     }
 
+    /**
+     * @param array{
+     *   ACL?: \AsyncAws\S3\Enum\ObjectCannedACL::*,
+     *   CacheControl?: string,
+     *   Metadata?: array<string, string>,
+     *   PartSize?: positive-int,
+     *   Concurrency?: positive-int,
```

Comment:
please use `int`, this syntax is not a standard, therefore not understood by all IDE.